.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: node_js
+
+node_js: "6"
+
+addons:
+  apt:
+    packages:
+      - xvfb
+      - fakeroot
+      - dpkg
+      - libdbus-1-dev
+      - libglib2.0-dev
+      - dbus
+
+before_install:
+  - cd debian/
```

Comment:
Oh, I already did that on my test branch in my fork. Here's the code for that:
```
os:
  - linux
  - osx
...

before_install:
  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then   cd debian/ ;       fi
  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then     cd darwin/;        fi
  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then     brew install yarn; fi
```

You have to make sure you install `yarn` for OSX, I was getting errors when building.
With this Travis willl run parallel builds for each OS.